### PR TITLE
Movie search

### PR DESCRIPTION
**Pull request by RetynaGirl**
This PR:
* Creates TMDB interaction service
* Creates MovieData class
* Creates movie search page and functionality  

All tests passing

### CODE REVIEW

File: spec/services/movie_api/tmdb_interaction_spec.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      ]
+
+      result = TMDBInteraction.create_movie_data(example_data)
+
+      expect(result[0]).to be_a(MovieData)
+
+      expect(result[0].id).to eq(400_160)
+    end
+  end
+
```
Comment (s-steel) [inline spec/services/movie_api/tmdb_interaction_spec.rb:49]: We'll add more tests once we learn more in class.

**Review by s-steel — APPROVED** [review]

File: app/services/movie_api/movie_data.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class MovieData
+  def initialize(data)
+    data.each do |key, value|
+      instance_variable_set("@#{key}", value)
+      self.class.send(:attr_accessor, key)
```
Comment (jdcarey128) [inline app/services/movie_api/movie_data.rb]: I think this should be changed to :attr_reader so as to limit outside modification.

**Review by jdcarey128 — CHANGES_REQUESTED** [review]
Quick change for MovieData class

File: app/services/movie_api/movie_data.rb (hunk 2 above)
Comment (RetynaGirl) [reply app/services/movie_api/movie_data.rb]: ohh yes definitely

**Review by s-steel — APPROVED** [review]

**Review by jdcarey128 — APPROVED** [review]
LGTM

**Pull request merged**